docs/source/community/governance.md

```diff
@@ -0,0 +1,279 @@
+*   Doing code reviews or commenting on technical documents
+*   Writing, editing, translating or reviewing the documentation
+*   Organizing events or evangelizing the project (e.g,. on Social Media)
+
+Contributors engage with the project through the issue tracker or by writing or editing documentation. They submit changes to the project itself via Pull Requests (PRs), which will be considered for inclusion in the project by existing maintainers (see next section).
+
+Contributors should follow the following guides when creating PRs:
+- [Contribution Process](https://docs.feast.dev/project/contributing)
+- [Development Guide](https://docs.feast.dev/project/development-guide).
+
+As contributors gain experience and familiarity with the project, their profile and commitment within the community will
+increase. At some stage, they may find themselves being nominated for being a maintainer.
+
+### Triagers
+Triagers are contributors who have shown that they are committed to Llama Stack's continued development by helping to
+manage the project's backlog. They are responsible for ensuring that the project's issues are well-organized and up to
+date so that contributors and maintainers can navigate the project efficiently and prioritize key requests from
+the community.
+
+### Code Owners
+
+Contributors who have shown that they are committed to Llama Stack's continued development through ongoing engagement
+but may not yet meet the requirements for maintainership can be added to the CODEOWNERS file to review and merge PRs under
+a specific domain.
+
+CODEOWNERS will generally be the first point of contacts in reviewing pull requests and will have commit privileges.
+
+### Maintainers (Project + Area)
+
+Maintainers are community members who have shown that they are committed to Llama Stack's continued development through
+ongoing engagement with the community. Because of this, maintainers have the right to merge PRs and have voting rights.
+
+> **Note**: maintainers, like other contributors, must make changes to Llama Stack via pull requests (with code review).
+> This applies to all changes to documentation, code, configuration, governance, and so on.
+
+Maintainers control the overall project organization, direction, and are responsible for resolving disputes. They also
+- Attend a regular maintainers sync
+- Participate in strategic planning, approve changes to the governance model, and manage the copyrights within the project outputs.
+- (optional) Plan the project roadmap and articulate the vision
+- (optional) Guide design decisions to reinforce key project values (e.g. simplicity)
+
+#### Optional maintainer responsibilities
+Other optional activities a maintainer (project or area maintainer) may participate in:
+  *   Perform code reviews for other maintainers and the community. The areas of specialization listed in [OWNERS.md](OWNERS.md) can be used to help with routing an issue/question to the right person.
```

Comment:
I don't think this should be optional.

Reply:
Agreed. 

I based this governance on Feast's and they had this section there.

I think the maintainers should decide what they feel is best and it can be transparent to the community so that expectations can be set well.